### PR TITLE
ci: enhance code benchmarks workflow

### PR DESCRIPTION
**Pull request by genryxy**
Part of artipie/artipie#431
Remove some lines for copying results because it is included into `make` step. Use custom actions for running benchmarks and creating PR

### CODE REVIEW

**Review by g4s8 — APPROVED** [review]
@genryxy thanks. Maybe it would be better to make GitHub action from benchmarks repo and include only some configuration in each adapter, like:
```yaml
- name: run benchmarks
  uses: artipie/benchmarks@latest
  with: 
    repo: conda-adapter
    tag: ${{ env.TAG }}
```

Then you can fix some bug only in one place, and it'll be easier to integrate, WDYT?

**Comment by genryxy** [comment]
@g4s8 yeah, it sounds reasonable. I am now on the way to reduce the number of boilerplate code. But creating such action would help to join all common parts

**Comment by genryxy** [comment]
@g4s8 could you please check. I used custom actions

File: .github/workflows/benchmarks.yml [hunk 1]
```diff
@@ -9,31 +9,11 @@ jobs:
...
-          git fetch origin
-          git switch -C master origin/master
-          git checkout -b bench-${{ env.TAG }}
-          git commit -m "bench: added results of benchmarks for ${{ env.TAG }}"
-          git push origin HEAD:bench-${{ env.TAG }}
+      - name: Run target benchmarks
+        uses: artipie/benchmarks@master
+        with:
+          target: "conda"
+          tag: "${GITHUB_REF#refs/*/}"
```
Comment (g4s8) [inline .github/workflows/benchmarks.yml:16]: Does it work here? Maybe it should be exported to some variable and then accessed by `${{ env.VARIABLE }}`?

**Review by g4s8 — APPROVED** [review]
If this code work, you can ignore my question and merge.

File: .github/workflows/benchmarks.yml (hunk 1 above)
Comment (genryxy) [reply .github/workflows/benchmarks.yml:16]: This [type](https://github.com/artipie/helm-adapter/blob/master/.github/workflows/benchmarks.yml#L16) of code works for Helm. But in the second type of action it does not work in Helm, therefore I made in the same way here

**Pull request merged**